ui/apps/platform/src/Containers/AccessControl/AccessControlBreadcrumbs.tsx

```diff
@@ -9,38 +9,24 @@ import { accessControlLabels } from 'messages/common';
 import { getEntityPath } from './accessControlPaths';
 
 export type AccessControlBreadcrumbsProps = {
-    entityType?: AccessControlEntityType;
+    entityType: AccessControlEntityType;
     entityName?: string;
-    isDisabled?: boolean;
-    isList?: boolean;
 };
 
 function AccessControlBreadcrumbs({
     entityType,
     entityName,
-    isDisabled,
-    isList,
 }: AccessControlBreadcrumbsProps): ReactElement {
-    let entityTypeBreadcrumb;
     const entityTypeLabel = entityType ? pluralize(accessControlLabels[entityType]) : null;
-    if (entityType) {
-        entityTypeBreadcrumb =
-            isDisabled || isList ? (
-                <BreadcrumbItem isActive>{entityTypeLabel}</BreadcrumbItem>
-            ) : (
-                <BreadcrumbItemLink to={getEntityPath(entityType)}>
-                    {entityTypeLabel}
-                </BreadcrumbItemLink>
-            );
-    }
 
     return (
         <>
-            <PageSection variant="light">
+            <PageSection variant="light" className="pf-u-py-md">
                 <Breadcrumb>
-                    <BreadcrumbItem isActive>Access Control</BreadcrumbItem>
-                    {entityTypeBreadcrumb}
-                    {entityName && <BreadcrumbItem isActive>{entityName}</BreadcrumbItem>}
```

Comment:
`isActive` is [ignored](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx#L45) on a non-link `BreadcrumbItem` when neither a custom container nor render prop are passed, so it was removed.